<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delete_person</class>
 <widget class="QWidget" name="delete_person">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>删除信息</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入你要删除对象的信息</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="classmate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>同学</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="collaege">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>同事</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="the_friend">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>朋友</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="relative">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>亲戚</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>他/她和你的关系：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
